packages/ns-plug/files/register

```diff
@@ -34,24 +34,16 @@ fi
 # Setup URLs
 case "$type" in
     community)
```

Comment:
There 3 checks of the type:
- one line 35
- one on line 47
- one on line 64

Please refactor the code to remove at least one check:
- we could merge the first and second one
- use only one operator type: if or switch